benches/centralized_telescope/number_steps.rs

```diff
@@ -14,7 +14,7 @@ use utils::{
             centralized::{benchmarks, BenchParam},
             Steps,
         },
-        test_vectors::{centralized::SHORT_TESTS, Data},
```

Comment:
It is easier to understand what Data is.
I would not mind shortening the name in internal files but not when it is user facing (tests, examples, benches and pub functions).